IFComp/root/src/entry/list.tt

```diff
@@ -54,7 +54,7 @@
     [% ELSE %]
         Not ready for entry yet (No game file)
     [% END %]
-    [% IF current_comp.status == 'open_for_judging' && ( entry.platform == 'inform-website' || entry.platform == 'parchment' ) %]
+    [% IF ( entry.platform == 'inform-website' || entry.platform == 'parchment' || entry.platform == 'quixe' ) %]
```

Comment:
Would it make sense to add a method to Entry objects, something like 'has_transcript' or 'supports_transcripts' or something like that, and use that method here instead?